<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_delete_book_ui</class>
 <widget class="QDialog" name="update_delete_book_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update &amp; Delete Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CBE6C7;</string>
  </property>
  <widget class="QComboBox" name="category_combobox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>category</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>quantity</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="update_value_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_book_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #D10000;
	border: 1px solid #000000;
    color: #CBE6C7;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #A30000;
}

QPushButton:pressed {
    background-color:#750000;
}
</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>New Value :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Member Id : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="text">
    <string>DELETE BOOK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="update_book_id_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="delete_book_id_entry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;
border-radius:10px;
border: 1px solid #122C4F;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="id_label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Book Id : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#122C4F;</string>
   </property>
   <property name="text">
    <string>Column :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="update_book_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #122C4F;
	border: 1px solid #000000;
    color: #CBE6C7;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #0D203A;
}

QPushButton:pressed {
    background-color: #091525;
}
</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="text">
    <string>UPDATE BOOK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
